pages/learn/develop/container-contracts.md

```diff
@@ -59,7 +59,7 @@ requires:
 
 __Note__ Any type other than `sw.os` or `sw.supervisor` will result in the contract being invalid.
 
-When deploying a release, if the contract requirements are not met, the release will fail, and all devices in the application will remain on their current release. This default behavior can be overridden by including an [optional label](#optional-contracts) in the application docker-compose.yml file. This label specifies which services with unmet requirements will be ignored, with the release being deployed to all other devices in the application. If there are any existing running containers for services with unmet requirements, they will not be destroyed and continue running the previous release.
+When deploying a release, if the contract requirements are not met, the release will fail, and all devices in the application will remain on their current release. This default behavior can be overridden by including an [optional label](#optional-containers) in the application docker-compose.yml file. This label specifies which services with unmet requirements will be ignored, with the release being deployed to all other devices in the application. If there are any existing running containers for services with unmet requirements, they will not be destroyed and continue running the previous release.
 
 Should a contract fail, the dashboard logs will contain detail about the failing contract:
```

Comment:
@CameronDiver will this fail over and over, or just the once?

Reply:
just thinking of people on cellular

Reply:
It will be rejected once every target poll, so no difference in terms of bandwidth